.github/workflows/release.yml

```diff
@@ -0,0 +1,45 @@
+name: release
+
+on:
+  release:
+    types: [published, edited]
+
+jobs:
+
+  upload-transifex:
+
+    name: Upload pot files to transifex
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+
+    - name: Install python dependencies
+      run: |
+        pip install numpy==1.17.4
```

Comment:
Not sure about this, just copy and paste from other jobs.

Reply:
There are some issues installing AiiDA when numpy is not already installed. Those are related to broken build setups of dependencies and should be mostly resolved, please see discussion on #3944 for example. In this case I suggest that we follow the [installation steps of the docs job](https://github.com/aiidateam/aiida-core/blob/d6e8c7d10537b754d1cb6334689ba2edf4347d43/.github/workflows/ci.yml#L7-L30) as part of the `continuous-integration` workflow (I assume that's what you did). At some point we can try to simplify the installation for the docs-related jobs, but that can be a separate PR.

Reply:
Thanks for explanation.